server/android/src/main/java/dev/slimevr/android/serial/AndroidSerialHandler.kt

```diff
@@ -43,12 +44,11 @@ class AndroidSerialHandler(val activity: AppCompatActivity) :
 	private var usbIoManager: SerialInputOutputManager? = null
 
 	private val listeners: MutableList<SerialListener> = CopyOnWriteArrayList()
-	private val getDevicesTimer = Timer("GetDevicesTimer")
-	private var watchingNewDevices = false
 	private var lastKnownPorts = setOf<SerialPortWrapper>()
 	private val manager = activity.getSystemService(Context.USB_SERVICE) as UsbManager
 	private var currentPort: SerialPortWrapper? = null
 	private var requestingPermission: String = ""
+	private var readBuffer: StringBuilder = StringBuilder(4096)
```

Comment:
```suggestion
	private var readBuffer: StringBuilder = StringBuilder(1024)
```
i think 1024 chars is enough?

Reply:
I originally set it to 512, but changed it to 4096 to match the desktop implementation. I could change it to 1024 though, maybe it would be better as that.